e2e/assertions/compareScreenshot.js

```diff
@@ -15,6 +16,27 @@ const ensureBaselinePhotoExists = (baselinePath, resultPath) => {
   }
 }
 
+const update = (baselinePath, resultPath, callback, data) => {
+  if (process.env.UPDATE_SCREENSHOTS === 'true') {
```

Comment:
I feel like it makes sense to assign the `process.env.UPDATE_SCREENSHOTS` globals to `const` variables instead of continually referring to them directly from `process`.